### ISSUE TITLE
get_task_settings('Sensitivities') does not include Effect, Cause and Secondary cause

### ISSUE THREAD

**Issue by pmendes** [copasi/basico#42]
Similar to [previous issue on get_task_settings](https://github.com/copasi/basico/issues/41#issue-2190791229), without this information it is not possible to know what the task will do. Moreover, it is not possible to set this task appropriately (I am assuming there is also no way to pass this information to set_task_settings)

**Comment by fbergmann** [comment]
For sensitivities there is set/ `get_sensitivity_settings()`

```yaml
{'scheduled': False,
 'update_model': False,
 'method': {'Delta factor': 0.001,
  'Delta minimum': 1e-12,
  'name': 'Sensitivities Method'},
 'report': {'filename': '',
  'report_definition': 'Sensitivities',
  'append': True,
  'confirm_overwrite': True},
 'sub_task': 'Steady State',
 'effect': 'Non-Constant Concentrations of Species',
 'cause': 'All Parameter Values',
 'secondary_cause': 'Not Set'}
```

an example is here <https://basico.readthedocs.io/en/latest/notebooks/Sensitivity_Analysis.html>.

**Comment by fbergmann** [comment]
The fix for #41 applies here to, and will let the CNs show up. I still think it would be better to use the specific sensitivity settings as I think they will be more convenient to use.

**Comment by pmendes** [comment]
Frank, I had not noticed get_sensitivity_settings() which should have worked well. But since the other fix also adds these then at least it looks more complete this way.  I'm marking it closed.
